.github/workflows/terraform.yml

```diff
@@ -70,6 +72,32 @@ jobs:
           soft_fail: true
           working_directory: "${{ matrix.module }}"
 
+      - name: Setup Go
+        uses: actions/setup-go@v4
+        with:
+          go-version: 1.19.4
```

Comment:
Since Go is being used as a test dependency, the version should probably be semver like `^1.19` or `^1`